packages/mobile/src/app/actions.ts

```diff
@@ -96,3 +117,20 @@ export const setAnalyticsEnabled = (enabled: boolean): SetAnalyticsEnabled => ({
   type: Actions.SET_ANALYTICS_ENABLED,
   enabled,
 })
+
+export const navigatePinProtected = (
+  routeName: string,
+  params?: NavigationParams
+): NavigatePinProtected => ({
+  type: Actions.NAVIGATE_PIN_PROTECTED,
+  routeName,
+  params,
+})
+
+export const startPinVerification = () => ({
```

Comment:
You should type these action creator to use the interface you defined above:
`export const startPinVerification = (): StartPinVerification => ({`